src/components/NavigationBar/Login/LoginNavigationBar.jsx

```diff
@@ -4,13 +4,10 @@ import OutlinedButton from '../../Button/Outlined';
 import { setInterfaceLanguage } from '../../../redux/reducer/interfaceLanguageSlice';
 import { useDispatch } from 'react-redux';
 import i18n from 'i18next';
-import Cookies from 'js-cookie';
 
 function NavigationBar() {
   const dispatch = useDispatch();
-  const [changeLanguageTo, setchangeLanguageTo] = useState(
```

Comment:
When a user is in the login screens ,the interface language shown is based on the selection of the user right (language button on the top right corner).

Reply:
Please don't implement the language cookie.  Ideally, when the App loads, and there is no "remember me" to automatically login the user and use the UI language set in their profile, then the i18n.language should be set to the browser language. This could be done with something like https://github.com/i18next/i18next-browser-languageDetector but please don't. The use case is rare and we have other priorities.  Sometimes I ask a question about why you changed something, but as you get to know me, when I ask a question it is to learn, and doesn't mean I want it done that way :-)